dp/generate_climos.py

```diff
@@ -134,19 +134,34 @@ def climo_outputs(time_resolution):
     # Apply per-variable processing
     # - Scale pr variable if it is not in desired units
     #   - TODO: Use UDUNITS here for more robust processing of units? It has a fairly heavy API. But this is uuuugly.
```

Comment:
What a delightful and elegant library! `pint` can't parse all valid `udunits` strings, but a simple transformation of the strings can fix that. 

Also the Python binding of `udunits2`, `cfunits`, only works in Python ver >=2.6 and <3.0, so we **must** use something different. I'm liking `pint`.